bnd-workspace/de.fraunhofer.abm.app/web/controllers/RegistrationController.js

```diff
@@ -32,9 +32,9 @@ function($rootScope, $scope, $http, Notification, $location){
 					self.available = false;
 				}
 			}, function(d){
-				Notification.error('Failed with ['+ d.status + '] '+ d.statusText);
+				Notification.error('Registration cannot be done at the moment. Please try agin later');
```

Comment:
Don't forget the punctuation: periods at the end of a sentence. 
Also correct in other places.